core/templates/dev/head/pages/base.html

```diff
@@ -129,8 +136,19 @@
         <div class="oppia-content-container">
           <div id="wrapper">
             <div class="oppia-main-body">
+              <nav class="navbar navbar-default oppia-navbar oppia-prevent-selection" role="navigation">
+                <div class="navbar-container">
+                  <top-navigation-bar></top-navigation-bar>
 
-              <top-navigation-bar></top-navigation-bar>
+                  <div class="collapse navbar-collapse ng-cloak">
+                    {% block navbar_breadcrumb %}
```

Comment:
Ah, I remember having some problems trying to use macros in directives in the past; it might have even been when trying to split this up. Working around this is going to be a bit tricky.